pkg/operator/staticpod/controller/installer/blame.go

```diff
@@ -0,0 +1,17 @@
+package installer
+
+import (
+	"fmt"
+	"strings"
+)
+
+// inspectInstallerReason inspects the reason given by the installer pod, which usually include the last log line from the installer pod
+// and try to point a finger on component that likely caused its failure to provide better operator message.
+func reasonWithBlame(reason string) string {
+	switch {
+	case strings.Contains(reason, "no route to host"):
+		return fmt.Sprintf("%s (check the network-operator for possible cause)")
```

Comment:
maybe even clearer like: most probably node networking is misbehaving

Reply:
updated